_contentTemplates/common/grid-treelist-autogenerated-columns.md

```diff
@@ -0,0 +1,4 @@
+#grid-treelist-autogenerated-columns-limitation
+When the component initializes, it expects to know what columns to render before the data comes.
+If the component is bound to a collection of dynamic objects (such as `Dictionary` or `ExpandoObject`) due to the nature of such objects and the fact that they can be changed at run time, the component cannot know what columns to render. Therefore, the `AutoGenerateColumns` feature is not applicable in such scenario and should be implemented manually (an example of such setup is available in our [public repository](https://github.com/telerik/blazor-ui/blob/master/grid/binding-to-expando-object/BindingToExpandoObject/Pages/AutoGeneratedColumns.razor)).
```

Comment:
The link to the public repository could have a more meaningful text, something like `an example is available in the [AutoGeneratedColumns page of the Binding a grid to ExpandoObject sample project](....)`

Reply:
Done with 0ba8870